### ISSUE TITLE
[Bug]: Password confirmation prompt opens, but input is disabled

### ISSUE THREAD

**Issue by RomanSixty** [piwik/piwik#21508]
### What happened?

When I try to add a new user I get a password confirmation prompt. However the input field has the `disabled` attribute so I can't confirm the action. At the moment I have to delete the `disabled` attribute via dev tools to add a user.

![matomo-password-disabled](https://github.com/matomo-org/matomo/assets/467921/ae385b01-cfd5-4b0d-9577-17ebba12d8de)

I tried to disable the additional password prompt via the workaround mentioned in #19904 but that didn't have an effect.

Also I have another instance running the same Matomo version, that doesn't have this problem. I compared code and config of both instances but didn't find the culprit. Does anyone have an idea of what can cause this? A password input field should probably never even be considered to be disabled.

### What should happen?

The prompt should not have a non-disabled password input.

### How can this be reproduced?

The prompt show up after (translated from German UI):

Administration -> System -> Users -> button "invite a new user" -> button "invite user"

### Matomo Version

Matomo 4

### Matomo Patch or Minor Version

4.15.1

### PHP Version

8.2

### Server Operating System

Debian GNU/Linux

### What browsers are you seeing the problem on?

Firefox, Chrome

### Computer Operating System

Windows

### Validations

- [X] Read our [Contributing Guidelines](https://github.com/matomo-org/matomo/blob/5.x-dev/CONTRIBUTING.md).
- [X] Follow our [Security Policy](https://github.com/matomo-org/matomo/blob/5.x-dev/SECURITY.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating duplicates.
- [X] The provided steps to reproduce is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the Bug.

**Comment by AltamashShaikh** [comment]
@RomanSixty Can you check if [this ](https://github.com/matomo-org/matomo/issues/21423#issuecomment-1772650974) applies to you as well ?

**Comment by RomanSixty** [comment]
No Login* plugins are installed, the default authentication of Matomo is in use.

**Comment by sgiehl** [comment]
@RomanSixty Would you mind checking the value of the javascript variable `piwik.requiresPasswordConfirmation`?
Simply open the browser console on the usermanagement page and paste that name in the console to receive it's value.

**Comment by RomanSixty** [comment]
@sgiehl The variable is `undefined`.

I noticed earlier that the password prompt can be left empty and still a user invitation is sent after submitting without credentials. However this is not very intuitive.

**Comment by sgiehl** [comment]
Interesting. It should be true or false 🤔 
Maybe check if this file contains the highlighted line on your installation: https://github.com/matomo-org/matomo/blob/4.x-dev/plugins/Morpheus/templates/_jsGlobalVariables.twig#L25

**Comment by RomanSixty** [comment]
@sgiehl Okay, that's a hint. I noticed a custom theme overrides `_jsGlobalVariables.twig` in this installation. That was probably done at a time when this line had not yet been introduced.

I deleted the file in the custom theme and now the variable is set to `false`. It's still the same behavior though.

**Comment by sgiehl** [comment]
Hm. It should be `true` by default, unless a plugin changes that. Could you maybe share the list of installed & activated plugins?

**Comment by RomanSixty** [comment]
Active are:

- Actions (Core) 
- Annotations (Core) 
- Contents (Core) 
- CustomDimensions (Core) 
- CustomJsTracker (Core) 
- Dashboard (Core) 
- DevicePlugins (Core) 
- DevicesDetection (Core) 
- Diagnostics (Core) 
- Ecommerce (Core) 
- Events (Core) 
- Feedback (Core) 
- GeoIp2 (Core)
- Goals (Core) 
- Heartbeat (Core) 
- ImageGraph (Core) 
- Insights (Core) 
- IntranetMeasurable (Core) 
- Live (Core) 
- Login (Core) 
- Marketplace (Core) 
- MobileMessaging (Core) 
- Monolog (Core) 
- MultiSites (Core) 
- Overlay (Core) 
- PagePerformance (Core) 
- PrivacyManager (Core) 
- ProfessionalServices (Core) 
- Referrers (Core) 
- Resolution (Core) 
- RssWidget (Core) 
- SEO (Core) 
- ScheduledReports (Core) 
- SegmentEditor (Core) 
- TagManager (Core) 
- Tour (Core) 
- TrackingSpamPrevention (v4.1.7) 
- Transitions (Core) 
- TwoFactorAuth (Core) 
- UserCountry (Core) 
- UserCountryMap (Core) 
- UserId (Core) 
- UserLanguage (Core) 
- VisitFrequency (Core) 
- VisitTime (Core) 
- VisitorInterest (Core) 
- VisitsSummary (Core) 
- Widgetize (Core) 

Installed but deactivated:

- CustomVariables (v4.1.3)
- DBStats (Core)
- MobileAppMeasurable (Core)
- Provider (v4.0.5)

**Comment by sgiehl** [comment]
Ok. So no plugins that would manipulate that 🤔 
Ist this method present in your files: https://github.com/matomo-org/matomo/blob/4.x-dev/core/Piwik.php#L291-L309
Otherwise I'm currently out of ideas where that problem might be...

**Comment by RomanSixty** [comment]
@sgiehl Yes, that method is present. And I guess that led me to the solution:

As mentioned in the beginning when faced with the disabled input I tried the workaround from #19904 to disable the whole confirmation prompt. That set the variable to `false`. Now that I removed that workaround the variable is `true` and the password input finally is no longer disabled. I'd like it better if there was no prompt at all, but at least now it's working as it should.

Thanks for your help.
